templates/monitoring/jobs/openshift_monitoring_federation.yaml

```diff
@@ -9,7 +9,8 @@
   metrics_path: /federate
   params:
     match[]:
-    - '{{ "{" }} endpoint="https-metrics" {{ "}" }}'
+    - 'kubelet_volume_stats_used_bytes{endpoint="https-metrics",namespace=~"redhat-rhmi-.*"}'
+    - 'kubelet_volume_stats_available_bytes{endpoint="https-metrics",namespace=~"redhat-rhmi-.*"}'
     - '{{ "{" }} service="kube-state-metrics" {{ "}" }}'
     - '{{ "{" }} service="node-exporter" {{ "}" }}'
     - '{{ "{" }} __name__=~"namespace_pod_name_container_name:.*" {{ "}" }}'
```

Comment:
I don't think these metrics need to be federated, since as far as I know this is replaced by `node_namespace_pod_container` in newer versions of Kubernetes/Prometheus. 

If that were the case, then this line in the 3scale dashboard would also need to be updated:
https://github.com/integr8ly/integreatly-operator/blob/ace4b90437c62ad2653b793ef300d0a26f7139a2/templates/monitoring/kube_state_metrics_3scale_alerts.yaml#L131

Reply:
removed the unused metric